tasks/query-cmr/test/query-granules.ts

```diff
@@ -167,36 +148,27 @@ describe('query#queryGranulesScrolling', function () {
       }
     });
 
-    it('uses a scrolling CMR search', function () {
-      expect(this.queryFields[0].scroll).to.equal('true');
+    it('does not use a scrolling CMR search', function () {
+      expect(this.queryFields[0].scroll).to.equal(undefined);
     });
-    it('does not use the page_size parameter', function () {
-      expect(this.queryFields[0].page_size).to.equal(undefined);
+    it('uses the page_size parameter', function () {
+      expect(this.queryFields[0].page_size).to.equal('2000');
     });
   });
 
   describe('when called with a max CMR granules limit', async function () {
-    hookQueryGranulesScrolling(1);
+    hookQueryGranules(1);
 
-    it('limits the STAC output', function () {
-      expect(this.result[1].length).to.equal(1);
+    it('sets the page_size parameter to the limit', function () {
+      expect(this.queryFields[0].page_size).to.equal('1');
     });
   });
 
   describe('when called with a max CMR granules limit that exceeds the number of CMR granules', async function () {
-    hookQueryGranulesScrolling(3000);
+    hookQueryGranules(3000);
 
-    it('the STAC output is not limitted', function () {
+    it('the STAC output is not limited', function () {
       expect(this.result[1].length).to.equal(3);
     });
   });
-
-  describe('when the CMR returns an error', async function () {
```

Comment:
I think we should still have a test where the CMR returns an error that the error message is returned.

Reply:
Done